libs/client/cornflow_client/constants.py

```diff
@@ -116,21 +116,25 @@
     ('iteration_limit', 'pyomo', 'cbc') : 'maxIt' ,
     ('solution_limit', 'pyomo', 'cbc') : 'maxSaved' ,
     ('time_limit', 'pyomo', 'cbc') : 'sec' ,
+    ('timeLimit', 'pyomo', 'cbc') : 'sec' ,
     ('pump_passes', 'pyomo', 'cbc') : 'pumpC' ,
     ('heuristics', 'pyomo', 'cbc') : 'heur' ,
 
     ('is_mip', 'pulp', 'cbc') : 'mip' ,
     ('abs_gap', 'pulp', 'cbc') : 'gapAbs' ,
     ('rel_gap', 'pulp', 'cbc') : 'gapRel' ,
     ('cutoff', 'pulp', 'cbc') : 'cuts' ,
-    ('time_limit', 'pulp', 'cbc') : 'timeLimit' ,
+    ('time_limit', 'pulp', 'cbc'): 'timeLimit',
+    ('timeLimit', 'pulp', 'cbc') : 'timeLimit' ,
     ('threads', 'pulp', 'cbc') : 'threads' ,
     ('presolve', 'pulp', 'cbc') : 'presolve' ,
+    ('msg', 'pulp', 'cbc'): 'msg',
```

Comment:
Some of the commas here have a space that I do not think is needed, but I do not want to drag this PR too much.